contracts/RelayRecipient.sol

```diff
@@ -3,81 +3,91 @@ pragma solidity >=0.4.0 <0.6.0;
+    function getRecipientBalance() public view returns (uint) {
+        return getRelayHub().balanceOf(address(this));
     }
 
-    function get_sender_from_data(address orig_sender, bytes memory msg_data) public view returns(address) {
-        address sender = orig_sender;
-        if (orig_sender == get_hub_addr() ) {
-            // At this point we know that the sender is a trusted RelayHubApi, so we trust that the last bytes of msg.data are the verified sender address.
+    function getSenderFromData(address origSender, bytes memory msgData) public view returns(address) {
+        address sender = origSender;
+        if (origSender == getHubAddr() ) {
+            // At this point we know that the sender is a trusted IRelayHub, so we trust that the last bytes of msg.data are the verified sender address.
             // extract sender address from the end of msg.data
-            sender = LibBytes.readAddress(msg_data, msg_data.length - 20);
+            sender = LibBytes.readAddress(msgData, msgData.length - 20);
         }
         return sender;
     }
 
-    function get_sender() public view returns(address) {
-        return get_sender_from_data(msg.sender, msg.data);
+    function getSender() public view returns(address) {
+        return getSenderFromData(msg.sender, msg.data);
     }
 
-    function get_message_data() public view returns(bytes memory) {
-        bytes memory orig_msg_data = msg.data;
-        if (msg.sender == get_hub_addr()) {
-            // At this point we know that the sender is a trusted RelayHubApi, so we trust that the last bytes of msg.data are the verified sender address.
+    function getMessageData() public view returns(bytes memory) {
+        bytes memory origMsgData = msg.data;
+        if (msg.sender == getHubAddr()) {
+            // At this point we know that the sender is a trusted IRelayHub, so we trust that the last bytes of msg.data are the verified sender address.
             // extract original message data from the start of msg.data
-            orig_msg_data = new bytes(msg.data.length - 20);
-            for (uint256 i = 0; i < orig_msg_data.length; i++)
+            origMsgData = new bytes(msg.data.length - 20);
+            for (uint256 i = 0; i < origMsgData.length; i++)
             {
-                orig_msg_data[i] = msg.data[i];
+                origMsgData[i] = msg.data[i];
             }
         }
-        return orig_msg_data;
+        return origMsgData;
     }
+    /*** Note :values below 10 are reserved by canRelay
+    *  @param encodedFunction the encoded function call (without any ethereum signature).
+    *  @param gasPrice - the gas price for this transaction
+    *  @param transactionFee - the relay compensation (in %) for this transaction
+    */
+    function acceptRelayedCall(address relay, address from, bytes memory encodedFunction, uint gasPrice, uint transactionFee, bytes memory approval) public view returns(uint);
```

Comment:
fix indent, newline after method (I almost commented that its a comment for the wrong method

Reply:
done